Makefile

```diff
@@ -1,5 +1,5 @@
 VERSION_DATE ?= $(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
-GO_FLAGS ?= GOOS=linux GOARCH=amd64 CGO_ENABLED=0 GO111MODULE=on
+GO_FLAGS ?= GOOS=linux GOARCH=s390x CGO_ENABLED=0 GO111MODULE=on
```

Comment:
We probably want a multi-arch image instead, like the one being proposed as part of #1203.